### PR TITLE
polish edit/update features

### PR DESCRIPTION
**Pull request by Garyguo2011**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ class User < ActiveRecord::Base
   attr_accessible :email, :password, :password_confirmation, :remember_me,
   					:first_name, :last_name
   # attr_accessible :title, :body
+  validates :email, :uniqueness => true
```
Comment (Garyguo2011) [inline app/models/user.rb]: Jinge please double check if email format is contain "@" and dot "."

**Comment by spdemille-ucb** [comment]
hey gary, i managed to install skype on my laptop which has a camera and
microphone :) also, if you pull the calcentral-task-view you can see the
CalCentral CSS applied to the task we already have :) i'm working on the
task view and trying to figure out how to do graphs. i will push more soon
:)

-- Sterling

**Comment by Garyguo2011** [comment]
Looks nice. Just need some changes for create/update/edit page. :D

Thanks for you effort.

Gary

-- 
Xinran Guo
xguo@berkeley.edu

On March 9, 2015 at 7:06:14 PM, spdemille-ucb (notifications@github.com) wrote:

hey gary, i managed to install skype on my laptop which has a camera and
microphone :) also, if you pull the calcentral-task-view you can see the
CalCentral CSS applied to the task we already have :) i'm working on the
task view and trying to figure out how to do graphs. i will push more soon
:)

-- Sterling

**Comment by spdemille-ucb** [comment]
what changes need to be done? the files look complete

**Comment by Garyguo2011** [comment]
Like we discuss last Friday, we probably need to keep edit/update in the same page of subtask view.
and probably a bit move compound around. like Remaining time etc.

Thanks
Gary

-- 
Xinran Guo
xguo@berkeley.edu

On March 9, 2015 at 7:47:21 PM, spdemille-ucb (notifications@github.com) wrote:

what changes need to be done? the files look complete

**Comment by Garyguo2011** [comment]
Hi Sterling,

Did you copy application.css as minimum file? (without any white space)


Thanks,
Gary

-- 
Xinran Guo
xguo@berkeley.edu

On March 9, 2015 at 8:11:56 PM, Xinran Guo (xguo@berkeley.edu) wrote:

Like we discuss last Friday, we probably need to keep edit/update in the same page of subtask view.
and probably a bit move compound around. like Remaining time etc.

Thanks
Gary

-- 
Xinran Guo
xguo@berkeley.edu

On March 9, 2015 at 7:47:21 PM, spdemille-ucb (notifications@github.com) wrote:

what changes need to be done? the files look complete

**Comment by spdemille-ucb** [comment]
hi gary, yes, it was without whitespace. it was already that way from the
cal-central website :) i have added a couple more changes if you pull
calcentral-task-view again. i'm still trying to figure out the best way to
do the update/create/delete links for the subtasks. i think i have
something soon :) thx!

**Comment by spdemille-ucb** [comment]
Hi Gary, I've added create and update subtasks to the calcentral-task-view
branch :) still working on delete

On Tue, Mar 10, 2015 at 2:12 AM, Sterling P DE MILLE <spdemille@berkeley.edu
> wrote:

> hi gary, yes, it was without whitespace. it was already that way from the
> cal-central website :) i have added a couple more changes if you pull
> calcentral-task-view again. i'm still trying to figure out the best way to
> do the update/create/delete links for the subtasks. i think i have
> something soon :) thx!
>
> On Mon, Mar 9, 2015 at 10:04 PM, Garyguo2011 <notifications@github.com>
> wrote:
>
>> Hi Sterling,
>>
>> Did you copy application.css as minimum file? (without any white space)
>>
>>
>> Thanks,
>> Gary
>>
>> --
>> Xinran Guo
>> xguo@berkeley.edu
>>

**Comment by Garyguo2011** [comment]
Nice, Thanks for the hard work.

Looks fantastic, Specially for the header bar. Super cool. Since you are helping (Daniel, and Roy)’s feature. Probably Your guys need to communicate with each other to avoid duplicate view work.

I figure out how to write about Rspec yesterday. will fill test cases correspond to every feature we come up with. i will check your branch. :D

Thanks,
Gary

-- 
Xinran Guo
xguo@berkeley.edu

On March 10, 2015 at 12:26:24 PM, spdemille-ucb (notifications@github.com) wrote:

Hi Gary, I've added create and update subtasks to the calcentral-task-view
branch :) still working on delete

On Tue, Mar 10, 2015 at 2:12 AM, Sterling P DE MILLE <spdemille@berkeley.edu
> wrote:

> hi gary, yes, it was without whitespace. it was already that way from the
> cal-central website :) i have added a couple more changes if you pull
> calcentral-task-view again. i'm still trying to figure out the best way to
> do the update/create/delete links for the subtasks. i think i have
> something soon :) thx!
>
> On Mon, Mar 9, 2015 at 10:04 PM, Garyguo2011 <notifications@github.com>
> wrote:
>
>> Hi Sterling,
>>
>> Did you copy application.css as minimum file? (without any white space)
>>
>>
>> Thanks,
>> Gary
>>
>> --
>> Xinran Guo
>> xguo@berkeley.edu
>>

**Comment by spdemille-ucb** [comment]
ok, yeah, i texted Roy about it and told him to copy what he needs :) i
just did a page view source and copy/paste lol :P

**Comment by spdemille-ucb** [comment]
i was trying to see if i could make the sub task and task view inside the
center column on the dashboard page of calcentral. i guess i can try
scanning the css document for the column width :)

On Tue, Mar 10, 2015 at 1:00 PM, Sterling P DE MILLE <spdemille@berkeley.edu
> wrote:

> ok, yeah, i texted Roy about it and told him to copy what he needs :) i
> just did a page view source and copy/paste lol :P
>
> On Tue, Mar 10, 2015 at 12:37 PM, Garyguo2011 <notifications@github.com>
> wrote:
>
>> Nice, Thanks for the hard work.
>>
>> Looks fantastic, Specially for the header bar. Super cool. Since you are
>> helping (Daniel, and Roy)’s feature. Probably Your guys need to communicate
>> with each other to avoid duplicate view work.
>>
>> I figure out how to write about Rspec yesterday. will fill test cases
>> correspond to every feature we come up with. i will check your branch. :D
>>
>> Thanks,
>> Gary
>>
>> --
>> Xinran Guo
>> xguo@berkeley.edu
>>
>> On March 10, 2015 at 12:26:24 PM, spdemille-ucb (notifications@github.com)
>> wrote:
>>
>> Hi Gary, I've added create and update subtasks to the calcentral-task-view
>> branch :) still working on delete
>>
>> On Tue, Mar 10, 2015 at 2:12 AM, Sterling P DE MILLE <
>> spdemille@berkeley.edu
>> > wrote:
>>
>> > hi gary, yes, it was without whitespace. it was already that way from
>> the
>> > cal-central website :) i have added a couple more changes if you pull
>> > calcentral-task-view again. i'm still trying to figure out the best way
>> to
>> > do the update/create/delete links for the subtasks. i think i have
>> > something soon :) thx!
>> >
>> > On Mon, Mar 9, 2015 at 10:04 PM, Garyguo2011 <notifications@github.com>
>> > wrote:
>> >
>> >> Hi Sterling,
>> >>
>> >> Did you copy application.css as minimum file? (without any white space)
>> >>
>> >>
>> >> Thanks,
>> >> Gary
>> >>
>> >> --
>> >> Xinran Guo
>> >> xguo@berkeley.edu
>> >>

**Comment by Garyguo2011** [comment]
Hi Sterling,

Wow. I saw that. that is super cool. I just noticed add a new field called END TIME. We didn’t add to DB yet right? Probably we can discuss about it before you add to DB.





-- 
Xinran Guo
xguo@berkeley.edu

On March 10, 2015 at 1:01:48 PM, spdemille-ucb (notifications@github.com) wrote:

i was trying to see if i could make the sub task and task view inside the
center column on the dashboard page of calcentral. i guess i can try
scanning the css document for the column width :)

On Tue, Mar 10, 2015 at 1:00 PM, Sterling P DE MILLE <spdemille@berkeley.edu
> wrote:

> ok, yeah, i texted Roy about it and told him to copy what he needs :) i
> just did a page view source and copy/paste lol :P
>
> On Tue, Mar 10, 2015 at 12:37 PM, Garyguo2011 <notifications@github.com>
> wrote:
>
>> Nice, Thanks for the hard work.
>>
>> Looks fantastic, Specially for the header bar. Super cool. Since you are
>> helping (Daniel, and Roy)’s feature. Probably Your guys need to communicate
>> with each other to avoid duplicate view work.
>>
>> I figure out how to write about Rspec yesterday. will fill test cases
>> correspond to every feature we come up with. i will check your branch. :D
>>
>> Thanks,
>> Gary
>>
>> --
>> Xinran Guo
>> xguo@berkeley.edu
>>
>> On March 10, 2015 at 12:26:24 PM, spdemille-ucb (notifications@github.com)
>> wrote:
>>
>> Hi Gary, I've added create and update subtasks to the calcentral-task-view
>> branch :) still working on delete
>>
>> On Tue, Mar 10, 2015 at 2:12 AM, Sterling P DE MILLE <
>> spdemille@berkeley.edu
>> > wrote:
>>
>> > hi gary, yes, it was without whitespace. it was already that way from
>> the
>> > cal-central website :) i have added a couple more changes if you pull
>> > calcentral-task-view again. i'm still trying to figure out the best way
>> to
>> > do the update/create/delete links for the subtasks. i think i have
>> > something soon :) thx!
>> >
>> > On Mon, Mar 9, 2015 at 10:04 PM, Garyguo2011 <notifications@github.com>
>> > wrote:
>> >
>> >> Hi Sterling,
>> >>
>> >> Did you copy application.css as minimum file? (without any white space)
>> >>
>> >>
>> >> Thanks,
>> >> Gary
>> >>
>> >> --
>> >> Xinran Guo
>> >> xguo@berkeley.edu
>> >>

**Comment by spdemille-ucb** [comment]
yeah, and i noticed a few things about making a task, like the status
should not be filled in be the user except for pre-determined values like
maybe a dropdown box :) we should all meet and discuss before class maybe

**Comment by spdemille-ucb** [comment]
oh and the end time was the timestamp we talked about for graphing purposes
:)

On Tue, Mar 10, 2015 at 2:37 PM, Sterling P DE MILLE <spdemille@berkeley.edu

**Comment by spdemille-ucb** [comment]
the "end time" or timestamp is NOT user edit-able :)

On Tue, Mar 10, 2015 at 2:38 PM, Sterling P DE MILLE <spdemille@berkeley.edu
> wrote:

> oh and the end time was the timestamp we talked about for graphing
> purposes :)
>
> On Tue, Mar 10, 2015 at 2:37 PM, Sterling P DE MILLE <

**Comment by spdemille-ucb** [comment]
we can hide that field for now too :)

On Tue, Mar 10, 2015 at 2:39 PM, Sterling P DE MILLE <spdemille@berkeley.edu
> wrote:

> the "end time" or timestamp is NOT user edit-able :)
>
> On Tue, Mar 10, 2015 at 2:38 PM, Sterling P DE MILLE <

**Pull request merged**